packages/form-js-editor/src/render/Renderer.js

```diff
@@ -22,6 +22,31 @@ export class Renderer {
   constructor(renderConfig, eventBus, formEditor, injector) {
     const { container, compact = false } = renderConfig;
 
+    eventBus.on('form.init', function () {
+      // emit <canvas.init> so dependent components can hook in
+      // this is required to register keyboard bindings
+      eventBus.fire('canvas.init', {
+        svg: container,
+        viewport: null,
+      });
+    });
+
+    // focus container on over if no selection
+    container.addEventListener('mouseover', function () {
+      if (document.activeElement === document.body) {
```

Comment:
We could / should use [`Canvas#restoreFocus()`](https://github.com/bpmn-io/diagram-js/blob/develop/lib/core/Canvas.js#L352) here.